File: test7Go/oop/models/pizza/HawaiianPizza.go
package pizza

import (
	"fmt"
)

type HawaiianPizza struct {
	SimplePizza
}

func NewHawaiianPizza() HawaiianPizza {
	simplePizza := NewSimplePizza("Hawaiien Pizza", 24.47)
	simplePizza.Topping = fmt.Sprintf("%s,pineapple,ham", simplePizza.Topping)

	return HawaiianPizza{simplePizza}
}

File: test7C#/test7/Model/Pizza/HawaiianPizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model.Pizza
{
    class HawaiianPizza : SimplePizza
    {
        public HawaiianPizza() : base("Hawaiien Pizza", Convert.ToDecimal(24.47))
        {
            base.Topping(base.Topping() + ",pineapple,ham");
        }
    }
}
